documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,233 @@
+= Quarkus Quickstart 
+
+This guide will show a quickstart how to create a Quarkus app and will briefly show the key functionalities that quarkus provides and how to start as a beginner or also experienced developers.
+
+
+== Quarkus 
```

Comment:
I think the heading Quarkus is more or less meaningless. Better call it Quarkus introduction or something like that